XML/XSD/Item.xsd

```diff
@@ -205,7 +205,7 @@
       <xs:element name="Physical" minOccurs="1" maxOccurs="1" type="items:Physical">
       </xs:element>
       <xs:element name="Equipment" minOccurs="0" maxOccurs="1" type="items:Equipment"/>
-      <xs:element name="StatModifiers" minOccurs="0" maxOccurs="1" type="items:StatModifiers"/>
+      <xs:element name="StatModifiers" minOccurs="0" maxOccurs="1" type="items:ItemStatModifiers"/>
```

Comment:
Should statmodifiers remain as `StatModifiers` and not `ItemStatModifiers` as things such as spells can also apply these effects?

Reply:
Note that the `type` changed here, not the element name. This is to prevent a name collision with `StatModifiers` which has a conflicting definition in `Castable.xsd`, cleanup of which is way outside this scope.